src/DonationForms/Blocks/DonationFormBlock/resources/editor/Edit.tsx

```diff
@@ -29,146 +28,55 @@ export default function Edit({clientId, attributes, setAttributes}: BlockEditPro
 
     const getDefaultFormId = useCallback(() => {
```

Comment:
Using `useCallback` function here is not necessary, actually, this will be more performant without using `useCallback`

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7082/commits/445155c66d0f359fe15b65683a1b0a81b731b8b6